metrics-micrometer/src/main/java/org/axonframework/micrometer/GlobalMetricRegistry.java

```diff
@@ -110,12 +129,55 @@ public Configurer registerWithConfigurer(Configurer configurer) {
         return NoOpMessageMonitor.instance();
     }
 
+    /**
+     * Registers new metrics to the registry to monitor a component of given {@code type}. The monitor will be
+     * registered with the registry under the given {@code name} and the default set of Micrometer {@link Tag}s.
+     * The default set of Micrometer {@link Tag}s includes the 'message payload type' and additionally the 'processor
+     * name'
+     * for the event processors.
+     * The returned {@link MessageMonitor} can be installed on the component to initiate the monitoring.
+     *
+     * @param componentType The type of component to register
+     * @param componentName The name under which the component should be registered to the registry
+     * @return MessageMonitor (with Micrometer {@link Tag}s/dimensions enabled) to monitor the behavior of a given type
+     *
+     * @throws IllegalArgumentException if the component type is not recognized
+     */
+    public MessageMonitor<? extends Message<?>> registerComponentWithDefaultTags(Class<?> componentType,
+                                                                                 String componentName) {
+        if (EventProcessor.class.isAssignableFrom(componentType)) {
+            return registerEventProcessor(EVENT_PROCESSOR_METRICS_NAME,
+                                          message -> Tags.of(TagsUtil.PAYLOAD_TYPE_TAG,
+                                                             message.getPayloadType().getSimpleName(),
+                                                             TagsUtil.PROCESSOR_NAME_TAG,
+                                                             componentName));
+        }
+        if (CommandBus.class.isAssignableFrom(componentType)) {
+            return registerCommandBus(componentName,
+                                      message -> Tags
```

Comment:
Nit: Might it be reasonable to add the following to the `TagsUtil`?:

```java
public static final Function<Message<?>, Tag> PAYLOAD_TYPE_TAGGER = message > Tags.of(PAYLOAD_TYPE_TAG, message.getPayloadType().getSimpleName());
```

Having some of these functions on the `TagsUtil` class would make it easier for users to pick and choose from the defaults we have. If they'd now want to set up their own monitor _with_ the defaults we have, they'll have to provide this line too.

Minor improvement of course, not that big a deal.

Reply:
Done